ebuild-writing/users-and-groups/text.xml

```diff
@@ -16,9 +16,10 @@ categories.
 First, check the <uri link="https://api.gentoo.org/uid-gid.txt">UID/GID
 assignment list</uri> and choose a free UID/GID in the range between
 101 and 499. If you are adding a user and a group with the same name,
-use the same number for their UID and GID, respectively. When in
-doubt,
-take the next free number from 499 downwards.
+use the same number for their UID and GID, respectively. When in doubt, take
+the next free number from 499 downwards. There is a helper script available in
+the data/api.git repository to aid finding the next free UID or GID. Run it by
+issuing <c>./bin/used_free_uidgids.sh</c> inside api.git repo.
```

Comment:
Strictly speaking, you cannot run a command in a repository. :smirk:
```suggestion
issuing <c>./bin/used_free_uidgids.sh</c> in the api.git checkout.
```

Reply:
Done! This wording is a bit harder for me to understand at least, but I guess it's technically more correct.